kw.sh

```diff
@@ -9,8 +9,28 @@ config_files_path=${config_files_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/etc"
 # Export external variables required by kworkflow
 export EASY_KERNEL_WORKFLOW
 
-complete -W "export e build b bi install i prepare p new n ssh s mail mount mo
-             umount um boot bo vars v up u codestyle c maintainers m help h" kw
+_autocomplete()
+{
+    local cur prev opts
```

Comment:
I In this case, I recommend you to use a large name that better describes these variables.
e.g., current_coomand, kw_options...